README.md

```diff
@@ -3,12 +3,12 @@
 This is the main source code repository for [Rust]. It contains the compiler,
 standard library, and documentation.
 
-[Rust]: https://www.rust-lang.org
+[Rust]: https://www.rust-lang.org/
 
-**Note: this README is for _users_ rather than _contributors_.
-If you wish to _contribute_ to the compiler, you should read the
-[Getting Started][gettingstarted] section of the rustc-dev-guide instead.
-You can ask for help in the [#new members Zulip stream][new-members].**
+**Note: this README is for _users_ rather than _contributors_. If you wish to
+_contribute_ to the compiler, you should read the
+[Getting Started][gettingstarted] section of the rustc-dev-guide instead. You
+can ask for help in the [#new members Zulip stream][new-members].**
```

Comment:
Markdown doesn't need orphan line starts like this: The previous way puts each individual topic on its own lines. That way diffs affect the appropriate lines and not irrelevant content.

Reply:
That isn't consistent with how other paragraphs in the Readme are formatted though. What makes these two different?

Reply:
Apologies for missing this question.

There's no need for excessive consistency. Alternatively, you can make them consistent in the more topic-oriented direction. Either a rule should be programmatically enforced, which I am fine with, or the formatting style should incline towards whatever is most natural, and thus will be most-preserved by subsequent edits.

I also didn't highlight **all** the line endings that I would not have changed this way, I merely did it to two examples to set a trend.